doc/unison-manual.tex

```diff
@@ -183,69 +197,11 @@
   Synchronization}, and by University of Pennsylvania's Institute for
 Research in Cognitive Science (IRCS).
 
-\SECTION{Installation}{install}{install}
-
-Unison is designed to be easy to install.  The following sequence of
-steps should get you a fully working installation in a few minutes.  If
-you run into trouble, you may find the suggestions on the
-\SHOWURL{http://www.cis.upenn.edu/\home{bcpierce}/unison/faq.html}{Frequently Asked
-Questions page} helpful.  Pre-built binaries are available for a
-variety of platforms.
-
-Unison can be used with either of two user interfaces:
-\begin{enumerate}
-\item a simple textual interface, suitable for dumb terminals (and
-running from scripts), and
-\item a more sophisticated graphical interface, based on Gtk2 (on
-       Linux/Windows) or the native UI framework (on OSX).
-\end{enumerate}
-
-You will need to install a copy of Unison on every machine that you
-want to synchronize.  However, you only need the version with a
-graphical user interface (if you want a GUI at all) on the machine
-where you're actually going to display the interface (the \CLIENT{}
-machine).  Other machines that you synchronize with can get along just
-fine with the textual version.
-
-
-\SUBSECTION{Downloading Unison}{download}
 
-See \ONEURL{https://github.com/bcpierce00/unison/wiki/Downloading-Unison}.
+\SECTION{Upgrading}{upgrading}
```

Comment:
One bug still slipped through. `\SECTION` requires three arguments. The third argument may be left empty. A non-empty third argument seems to make this section a "topic" in the UI (seen by `unison -doc topic` and similarly in the GUI).